src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: YouTube enhances UX with Media Capabilities API
+subhead: |
+  Ensure buttery smooth playback when your users are on different devices.
+authors:
+  - beaufortfrancois
+date: 2019-06-05
+hero: hero.jpeg
+hero_position: top
+alt: Youtube application screengrab
+description: |
+  In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase
+  in MTBR with only a 0.4% decrease in average resolution of videos served.
+tags:
+  - post
+  - media
+  - ux
+  - case-study
+---
+
+In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase in
```

Comment:
```suggestion
In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase in
```
I think this is supposed to be an abstract for the post, but I worry that it won't be clear without context. We mention a new API and drop a likely unfamiliar term (MTBR) without having established why those things are relevant to the reader. I'd suggest dropping this sentence and the following aside and just jumping into the problem.

Reply:
Done.